std/xml.d

```diff
@@ -1059,9 +1059,13 @@ class Tag
                 munch(s,whitespace);
                 reqc(s,'=');
                 munch(s,whitespace);
-                reqc(s,'"');
-                string val = decode(munch(s,"^\""), DecodeMode.LOOSE);
-                reqc(s,'"');
+                string atr_val_put_chars="'\"";   //chars in which tags can be put into
+                char atr_put_into = reqs(s,atr_val_put_chars);
```

Comment:
I think there's no need for `atr_val_put_chars` anymore, as it's only used once. Also, the name `atr_put_into` doesn't make a lot of sense to me. How about `quote` instead? => `char quote = reqs(s,"'\"");`